CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 6.5.0 (Apr 15, 2022). Tested on Artifactory 7.37.13
+
+IMPROVEMENTS:
+
+* Resources added for Pub package type of Local Repository [GH-409]
```

Comment:
Make sure this ticket number is linkable in the readme

Reply:
are you proposing like following.

[GH-409](https://github.com/jfrog/terraform-provider-artifactory/pull/409)

or #408  https://github.com/jfrog/terraform-provider-artifactory/issues/408

Reply:
done